app-web/__tests__/utils/search.test.js

```diff
@@ -0,0 +1,44 @@
+import { getSearchResults } from '../../src/utils/search';
```

Comment:
moved search utils to its own file, the tests were moved as well. no changes made to tests.